src/drivers/rt1020.c

```diff
@@ -52,16 +52,16 @@ uint8_t s_txbuf[ETH_DESC_CNT][ETH_PKT_SIZE] __attribute__((aligned((64U))));
 
 static struct mg_tcpip_if *s_ifp;  // MIP interface
 
-enum { PHY_ADDR = 2, PHY_BCR = 0, PHY_BSR = 1, PHY_PC1R = 30, PHY_PC2R = 31 };
+enum { PHY_BCR = 0, PHY_BSR = 1, PHY_ID2 = 2, PHY_ID3 = 3 };
 
-static uint32_t eth_read_phy(uint8_t addr, uint8_t reg) {
+static uint16_t rt10xx_phy_read(uint8_t addr, uint8_t reg) {
```

Comment:
Most drivers have an "eth_write_phy"
Yes, the new name is taxonomy friendlier, I like it

Reply:
I was thinking that in (very unlikely situation) we have 2 drivers enabled - they  would clash then.